### ISSUE TITLE
2023 01 03 - Marketing Review - SOGS

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/state-of-green-software#84]
@NAMRATA-WOKE now that we are no longer promoting the survey, I'm simplifying the weekly marketing review tickets. In this ticket, I'm sharing with you all the info I have about SOGS. **Please review and let me know if you need anything else to be able to give the heads-up to org leads about this.**
- ETA of the SOGS report: **April (might change)**
- Format: website and downloadable PDF
- The PDF is just a regular report, nothing fancy.
- The website is explained [in this doc](https://docs.google.com/document/d/1WKzL20AGOumKn6-P3YFxdv0j7fUEcdz5VKY1LNrYEdo/edit#) and [in this video](https://www.loom.com/share/133d98744844467286751c625cb74fe4). Basically the website should be driven by the insights/findings from the report. Each insight can be viewed on it's own without reading the rest of the report, this means we will have to repeat content. You can go through the insights in any order, read the ones that you are interested in and ignore the rest.
- We might need your input / advice on what titles to use for the insights. We'll reach out when the time comes.

---
_Resources: [Marketing assets & useful links](https://github.com/Green-Software-Foundation/state-of-green-software/issues/41), [SOGS KPIs](https://docs.google.com/spreadsheets/d/1x_R9Z0qIn7WGvSA_tuW2QHiJIOBjlR0OhjzuLg6aFFk/edit#gid=0), [ShortIO Conversion rates](https://docs.google.com/spreadsheets/d/1yYhCH8hvC-pSbJ0RZVkIPzWzMOE41VT5MKj1PQoexVA/edit#gid=0) (last updated on Dec 12)_

**Comment by NAMRATA-WOKE** [comment]
Thanks @holanita. Will take a look at the website docs and let you know if I have anything to contribute at this stage. Do we have a sense of when the test site will be ready? Do we have an editor and proofreader lined up for the report?

**Comment by holanita** [comment]
I don't have an ETA for the website yet. We will need an editor for sure, just need to figure out whether it will be done by Urs or Catherine (depends on their availability).

**Comment by holanita** [comment]
@NAMRATA-WOKE changed the ETA to April. Might be delayed even more. No need for any promo activity atm I believe.

**Comment by zanete** [comment]
capturing a conversation on mktg bi-weekly call: 
@NAMRATA-WOKE needs clarity on how we want to amplify and promote the report. There are lots of things that can be done:
* do a webinar across two timezones to mx reach
* do an interactive website vs static
* need to write blog articles about what's in the report to help digest
* video content for promotion
* Press release
etc, all of these take time and the estimated time is no less than 2 months, especially if we want to leverage partners.
Media outreach for earned coverage is event more time consuming.

In summary, what is the likelyhood that this report will be produced by April, and is it start of April or end of April.
The more we can share the better, then Namrata doesn't have to come back in and frequently check and edit plans as that is time and money wasted. 
cc @jawache @holanita

**Comment by zanete** [comment]
@holanita sorry for missing a question mark after the question, which might have been missed, so just in case I will repeat it. If you didn't miss it and have planned to write a response, then please ignore this comment.

> what is the likelyhood that this report will be produced by April, and is it start of April or end of April?

**Comment by holanita** [comment]
@zanete the project is on hold, @jawache asked me to stop working on it. he is having some discussions with the chairs and stakeholders in the background, please ask him

**Comment by NAMRATA-WOKE** [comment]
@jawache what's the status on SOGS? It's a good idea for us to send an update to marketing/comms teams at member organizations, especially if we want them to amplify / engage leaders in any way.

**Comment by jawache** [comment]
@NAMRATA-WOKE we've  just hired a new researcher, the goal is to get something out in 6w - 8w. Because of where we are with the project it's just about getting it over the line, we don't have a PM on it yet and until we do let's not give any firm dates to anyone. I'm comfortable sharing rough timeline with the caveat it can change and we won't get a firm date until much closer to release (because no PM, people are just going to work on it until they say it's kinda done)

RE: Keeping orgs upto date, can you come up with a proposal for a monthly email that goes out to the marketing/comms team members? I'd prefer an approach where we list out our projects, next milestones, how close we are to the next milestone, when we're likely to launch and with what probability. Some blurb for each one maybe, you know this audience and what they need.
